fastlane/lib/fastlane/actions/current_build_number.rb

```diff
@@ -0,0 +1,142 @@
+                                       short_option: "-k",
+                                       env_name: "CURRENT_BUILD_NUMBER_LIVE_TEAM_ID",
+                                       description: "The ID of your iTunes Connect team if you're in multiple teams",
+                                       optional: true,
+                                       is_string: false, # as we also allow integers, which we convert to strings anyway
+                                       default_value: CredentialsManager::AppfileConfig.try_fetch_value(:itc_team_id),
+                                       verify_block: proc do |value|
+                                         ENV["FASTLANE_ITC_TEAM_ID"] = value.to_s
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :live,
+                                       short_option: "-l",
+                                       env_name: "CURRENT_BUILD_NUMBER_LIVE",
+                                       description: "Query the live version (ready-for-sale)",
+                                       optional: true,
+                                       default_value: true),
+          FastlaneCore::ConfigItem.new(key: :version,
+                                       env_name: "LATEST_VERSION",
+                                       description: "The version number whose latest build number we want",
+                                       optional: true),
+          FastlaneCore::ConfigItem.new(key: :team_name,
+                                       short_option: "-e",
+                                       env_name: "LATEST_TESTFLIGHT_BUILD_NUMBER_TEAM_NAME",
+                                       description: "The name of your iTunes Connect team if you're in multiple teams",
+                                       optional: true,
+                                       default_value: CredentialsManager::AppfileConfig.try_fetch_value(:itc_team_name),
+                                       verify_block: proc do |value|
+                                         ENV["FASTLANE_ITC_TEAM_NAME"] = value.to_s
+                                       end)
+        ]
+      end
+
+      def self.output
+        [
+          ['LATEST_BUILD_NUMBER', 'The latest build number of either live or testflight version']
+        ]
+      end
+
+      def self.details
+        "Returns the current build number of either the live or testflight version - it is usefull for getting the build_number of the current, ready-for-sale, app version, it also works on non-live testflight version. If you need to handle more build-trains please see `latest_testflight_build_number`"
+      end
+
+      def self.example_code
+        [
+          'current_build_number'
+        ]
+      end
+
+      def self.authors
+        ["hjanuschka"]
+      end
+
+      def self.category
+        :misc
+      end
+
+      def self.is_supported?(platform)
+        true
```

Comment:
`is_supported?` is supposed to return true for the action target's platform - so in this case `[:ios, :mac].include?(platform)` is probably more appropriate